examples/python/uart_sender.py

```diff
@@ -21,6 +21,10 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
+#
+# Example Usage: Sends data through UART and expects `X` flag as response from
```

Comment:
This description is incorrect, `_sender` doesn't expect to receive any flags, it *sends* a bunch of stuff, including a flag and then just reads and displays whatever data it receives after that.

Reply:
Err. This was my bad. How about, `Sends some info messages along with X flag and expects a response from the other end. This example requires uart_receiver.py to be running on the other end`?